src/habitat.cr

```diff
@@ -70,8 +72,8 @@ class Habitat
   end
 
   module SettingsHelpers
-    macro setting(decl)
-      {% HABITAT_SETTINGS << decl %}
+    macro setting(decl, example = Nil)
```

Comment:
I think you could do `"some_value"` here and then remove the default in the MissingSettingError